docs/src/main/asciidoc/contributing.adoc

```diff
@@ -131,6 +131,7 @@ $ touch .springformat
 ==== Intellij IDEA
 
 In order to setup Intellij you should import our coding conventions, inspection profiles and set up the checkstyle plugin.
+The following files can be found in the https://raw.githubusercontent.com/spring-cloud/spring-cloud-build/master/spring-cloud-dependencies-parent/eclipse-code-formatter.xml[Spring Cloud Build] project.
```

Comment:
Maybe https://github.com/spring-cloud/spring-cloud-build/tree/master/spring-cloud-build-tools is a better link?

Reply:
Fixed. Sorry I copied the link used above in the doc.